audit_config.yaml

```diff
@@ -16,7 +16,14 @@ S3_PUBLIC_POLICY_GETOBJECT_ONLY:
   severity: Info
   is_global: True
   group: S3
-
+  
+S3_ENCRYPTION:
+  title: s3 encryption is not enabled
+  description: encryption is not done on these buckets
```

Comment:
I would reword to "S3 bucket encryption is not enabled"

Reply:
done

Reply:
`description` isn't as explanatory as the other S3 checks, Scott probably has a better idea of what the description should be than I do 🤔